boa/src/vm/opcode.rs

```diff
@@ -729,6 +800,7 @@ impl Opcode {
             Opcode::Pop => "Pop",
             Opcode::Dup => "Dup",
             Opcode::Swap => "Swap",
+            Opcode::Swap2 => "Swap2",
```

Comment:
Do we want a more descriptive name for this? "SwapFirstAndThird" or something?

Reply:
Also could you give some explanation of why this is needed? Im sure its fine i just couldn't wrap my head around the examples its being used in. Thanks

Reply:
This was only needed for one specific thing.
Like @HalidOdat commented, I will change the `DefineOwnPropertyByValue` opcode and remove `Swap2`. This was leftover, because I implemented the `Define*` opcodes for object literals later. The previous `Set*` opcodes are used in other places, so I did not want to duplicate those.